### PR TITLE
update code-block formatting changes for playbook_variables.rst

### PR DESCRIPTION
**Pull request by Accoustium**
formatting updates for https://github.com/ansible/ansible/issues/75899

### CODE REVIEW

File: docs/docsite/rst/user_guide/playbooks_variables.rst [hunk 1]
```diff
@@ -49,14 +49,18 @@ Simple variables combine a variable name with a single value. You can use this s
...
 
   remote_install_path: /opt/my_app_config
 
 Referencing simple variables
 ----------------------------
 
-After you define a variable, use Jinja2 syntax to reference it. Jinja2 variables use double curly braces. For example, the expression ``My amp goes to {{ max_amp_value }}`` demonstrates the most basic form of variable substitution. You can use Jinja2 syntax in playbooks. For example::
+After you define a variable, use Jinja2 syntax to reference it. Jinja2 variables use double curly braces. For example, the expression ``My amp goes to {{ max_amp_value }}`` demonstrates the most basic form of variable substitution. You can use Jinja2 syntax in playbooks. For example:
+
+.. code-block:: yaml
```
Comment (ariordan-redhat) [inline docs/docsite/rst/user_guide/playbooks_variables.rst]:
```suggestion
.. code-block:: yaml+jinja
```

File: docs/docsite/rst/user_guide/playbooks_variables.rst [hunk 2]
```diff
@@ -75,13 +79,17 @@ When to quote variables (a YAML gotcha)
 
 If you start a value with ``{{ foo }}``, you must quote the whole expression to create valid YAML syntax. If you do not quote the whole expression, the YAML parser cannot interpret the syntax - it might be a variable or it might be the start of a YAML dictionary. For guidance on writing YAML, see the :ref:`yaml_syntax` documentation.
 
-If you use a variable without quotes like this::
+If you use a variable without quotes like this:
+
+.. code-block:: yaml
```
Comment (ariordan-redhat) [inline docs/docsite/rst/user_guide/playbooks_variables.rst]:
```suggestion
.. code-block:: yaml+jinja
```

File: docs/docsite/rst/user_guide/playbooks_variables.rst [hunk 3]
```diff
@@ -75,13 +79,17 @@ When to quote variables (a YAML gotcha)
...
 
     - hosts: app_servers
       vars:
           app_path: {{ base_path }}/22
 
-You will see: ``ERROR! Syntax Error while loading YAML.`` If you add quotes, Ansible works correctly::
+You will see: ``ERROR! Syntax Error while loading YAML.`` If you add quotes, Ansible works correctly:
+
+.. code-block:: yaml
```
Comment (ariordan-redhat) [inline docs/docsite/rst/user_guide/playbooks_variables.rst]:
```suggestion
.. code-block:: yaml+jinja
```

File: docs/docsite/rst/user_guide/playbooks_variables.rst [hunk 4]
```diff
@@ -107,7 +117,9 @@ You can define variables with multiple values using YAML lists. For example::
 Referencing list variables
 --------------------------
 
-When you use variables defined as a list (also called an array), you can use individual, specific fields from that list. The first item in a list is item 0, the second item is item 1. For example::
+When you use variables defined as a list (also called an array), you can use individual, specific fields from that list. The first item in a list is item 0, the second item is item 1. For example:
+
+.. code-block:: yaml
```
Comment (ariordan-redhat) [inline docs/docsite/rst/user_guide/playbooks_variables.rst]:
```suggestion
.. code-block:: yaml+jinja
```

File: docs/docsite/rst/user_guide/playbooks_variables.rst [hunk 5]
```diff
@@ -174,11 +192,15 @@ Registered variables are host-level variables. When you register a variable in a
 Referencing nested variables
 ============================
 
-Many registered variables (and :ref:`facts <vars_and_facts>`) are nested YAML or JSON data structures. You cannot access values from these nested data structures with the simple ``{{ foo }}`` syntax. You must use either bracket notation or dot notation. For example, to reference an IP address from your facts using the bracket notation::
+Many registered variables (and :ref:`facts <vars_and_facts>`) are nested YAML or JSON data structures. You cannot access values from these nested data structures with the simple ``{{ foo }}`` syntax. You must use either bracket notation or dot notation. For example, to reference an IP address from your facts using the bracket notation:
+
+.. code-block:: yaml
```
Comment (ariordan-redhat) [inline docs/docsite/rst/user_guide/playbooks_variables.rst]:
```suggestion
.. code-block:: yaml+jinja
```

File: docs/docsite/rst/user_guide/playbooks_variables.rst [hunk 6]
```diff
@@ -174,11 +192,15 @@ Registered variables are host-level variables. When you register a variable in a
...
 
     {{ ansible_facts["eth0"]["ipv4"]["address"] }}
 
-To reference an IP address from your facts using the dot notation::
+To reference an IP address from your facts using the dot notation:
+
+.. code-block:: yaml
```
Comment (ariordan-redhat) [inline docs/docsite/rst/user_guide/playbooks_variables.rst]:
```suggestion
.. code-block:: yaml+jinja
```

File: docs/docsite/rst/user_guide/playbooks_variables.rst [hunk 7]
```diff
@@ -275,7 +303,9 @@ JSON string format
     ansible-playbook release.yml --extra-vars '{"version":"1.23.45","other_variable":"foo"}'
     ansible-playbook arcade.yml --extra-vars '{"pacman":"mrs","ghosts":["inky","pinky","clyde","sue"]}'
 
-When passing variables with ``--extra-vars``, you must escape quotes and other special characters appropriately for both your markup (for example, JSON), and for your shell::
+When passing variables with ``--extra-vars``, you must escape quotes and other special characters appropriately for both your markup (for example, JSON), and for your shell:
+
+.. code-block:: text
```
Comment (ariordan-redhat) [inline docs/docsite/rst/user_guide/playbooks_variables.rst]:
```suggestion
.. code-block:: shell
```

**Review by ariordan-redhat — CHANGES_REQUESTED** [review]
Adding suggestions for code-block lexers

**Comment by ariordan-redhat** [comment]
@Accoustium 
Thanks for creating the PR.
I made suggestions for some lexer values.

File: docs/docsite/rst/user_guide/playbooks_variables.rst (hunk 6 above)
Comment (Accoustium) [reply docs/docsite/rst/user_guide/playbooks_variables.rst]: Updated with suggested change

File: docs/docsite/rst/user_guide/playbooks_variables.rst (hunk 5 above)
Comment (Accoustium) [reply docs/docsite/rst/user_guide/playbooks_variables.rst]: Updated with suggested change

File: docs/docsite/rst/user_guide/playbooks_variables.rst (hunk 1 above)
Comment (Accoustium) [reply docs/docsite/rst/user_guide/playbooks_variables.rst]: Updated with suggested change

File: docs/docsite/rst/user_guide/playbooks_variables.rst (hunk 2 above)
Comment (Accoustium) [reply docs/docsite/rst/user_guide/playbooks_variables.rst]: Updated with suggested change

File: docs/docsite/rst/user_guide/playbooks_variables.rst (hunk 3 above)
Comment (Accoustium) [reply docs/docsite/rst/user_guide/playbooks_variables.rst]: Updated with suggested change

File: docs/docsite/rst/user_guide/playbooks_variables.rst (hunk 4 above)
Comment (Accoustium) [reply docs/docsite/rst/user_guide/playbooks_variables.rst]: Updated with suggested change

File: docs/docsite/rst/user_guide/playbooks_variables.rst (hunk 5 above)
Comment (Accoustium) [reply docs/docsite/rst/user_guide/playbooks_variables.rst]: Updated with suggested change

**Comment by Accoustium** [comment]
Thank you for the suggestions.  I did notice they were Jinja formats, but was unsure if you wanted jinja to be referenced or just the standard file type additions.

**Pull request merged**